setup.py

```diff
@@ -45,7 +49,7 @@ def run_tests(self):
     name='thoth-adviser',
     version=get_version(),
     description='Package and package stack adviser for the Thoth project',
-    long_description='Package and package stack adviser for the Thoth project',
+    long_description=read('README.rst'),
```

Comment:
What about something like:

```python
from pathlib import Path

# ...
long_description=Path('README.rst').read_text()
# ...
```

Reply:
I dont know, I took the change straight from the pypi docs :)